sources/Algo/include/Algo.h

```diff
@@ -236,13 +236,15 @@ class LoadDefinitionAlgorithm : public NodeAlgorithm {
   /**
    * @brief Perform the algorithm
    *
-   * Update the list with the loads of the node
+   * Update the list with the loads of the node. Only add the loads for which the nominal voltage of the node is 
+   * superior to dsoVoltageLevel, otherwise we use the default modelica library
```

Comment:
For me it is not relevant to say that we use the default modelica model. This is the dynawo behaviour. The comments should only state that load is ignored if criteria is not passed

Reply:
moreover it is not a modelica library, we can split the apple in 2 and says that "otherwise the load is ignored and is handled by the default dynawo behavior"